### PR TITLE
developer: Add support to reset accessibility settings to default

### PR DESCRIPTION
**Pull request by harshitshah4**
## For community

⬇️  **Please click the 👍 reaction instead of leaving a `+1` or 👍  comment**

### CODE REVIEW

File: pymobiledevice3/cli/developer.py [hunk 1]
```diff
@@ -841,6 +841,15 @@ def accessibility_settings_set(service_provider: LockdownClient, setting, value)
     service.set_setting(setting, eval(value))
     OSUTILS.wait_return()
 
+@accessibility_settings.command('reset', cls=Command)
+def accessibility_settings_set(service_provider: LockdownClient):
```
Comment (doronz88) [inline pymobiledevice3/cli/developer.py]:
```suggestion
def accessibility_settings_reset(service_provider: LockdownClient):
```

File: pymobiledevice3/cli/developer.py [hunk 2]
```diff
@@ -841,6 +841,15 @@ def accessibility_settings_set(service_provider: LockdownClient, setting, value)
...
+    """
+    reset accessibility settings to default
+
+    in order to list all available use the "show" command
```
Comment (doronz88) [inline pymobiledevice3/cli/developer.py]: remove this line as this isn't relevant to this command as an argument

**Review by doronz88 — CHANGES_REQUESTED** [review]

File: pymobiledevice3/cli/developer.py (hunk 2 above)
Comment (harshitshah4) [reply pymobiledevice3/cli/developer.py]: Done in [this](https://github.com/doronz88/pymobiledevice3/pull/1297/commits/dea8673f662ca1b0bda58293dfa84004358aa071) commit.

File: pymobiledevice3/cli/developer.py (hunk 1 above)
Comment (harshitshah4) [reply pymobiledevice3/cli/developer.py]: Done in [this](https://github.com/doronz88/pymobiledevice3/pull/1297/commits/dea8673f662ca1b0bda58293dfa84004358aa071) commit

**Review by doronz88 — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**